sdk/aqueduct/artifacts/generic_artifact.py

```diff
@@ -59,10 +51,16 @@ def get(self, parameters: Optional[Dict[str, Any]] = None) -> Any:
             InternalServerError:
                 An unexpected error occurred in the server.
         """
+        if self._is_content_deleted():
+            return None
```

Comment:
I think it's important to point out to the reader that execution status can never be DELETED (since we condition on status later in the method) after passing this check.. Can we make that an assertion after each of these?